server/utils/multerUpload.js

```diff
@@ -1,15 +1,6 @@
 import multer from "multer";
 
-const multerStorage = multer.diskStorage({
-  destination: (req, file, nextFn) => {
-    nextFn(null, "public/img/users");
-  },
-  filename: (req, file, nextFn) => {
-    // user-id-currentTimestamp.jpeg
-    const ext = file.mimetype.split("/")[1];
-    nextFn(null, `user-${req.user.id}-${Date.now()}.${ext}`);
-  }
-});
+const multerStorage = multer.memoryStorage();
```

Comment:
I think we want to use s3 storage right as opposed to memory storage? This will be important when we deploy

Reply:
@skassam21 oh yeah sorry my bad. will fix! thanks!